fix(nimbus): harden fenix integration test against intermittent RS-fetch race

## Context

The Fenix Enrollment integration test (added in #15345) has an intermittent failure on the release matrix job. Example: run [24788521941](https://github.com/mozilla/experimenter/actions/runs/24788521941) on PR #15405 — beta passed, release failed with `AssertionError: No log-state row found for fenix-release-integration-test` and the logcat dump shows `long-term-holdback-2025-h1-growth-android | no-feature-fenix | delivery` enrolled instead.

## The race

Fenix release/beta builds call `maybeFetchExperiments()` on startup against the production Remote Settings server. Our test does this sequence:

1. `adb shell svc wifi disable`
2. `adb shell svc data disable`
3. `nimbus-cli enroll ...` (launches Fenix with the enroll intent)
4. Fenix starts, runs `NimbusBuilder`, fires a background fetch of prod RS
5. `HomeActivity.onCreate` → `initializeTooling(intent)` resets DB, applies our experiment locally, sets fetch-enabled=false
6. **If** the background fetch from step 4 completes before `setFetchEnabled(false)` — it lands with the prod experiment list (which doesn't contain our test experiment)
7. Next evolve: our experiment isn't in the fresh list → treated as server-ended → unenrolled
8. A prod experiment claiming `no-feature-fenix` (same feature slot as our test) enrolls instead
9. log-state dumps that row, assertion fails

The `svc wifi/data disable` pair returns immediately but the emulator's network stack takes a moment to actually tear down. If Fenix establishes the RS socket in that window (cached DNS, in-flight connection), the fetch completes before the network is fully down.

## Fix

Swap the two `svc` commands for one `adb shell cmd connectivity airplane-mode enable` (Android 10+). `cmd connectivity` returns after the broadcast has propagated and radios are down — it's the synchronous equivalent of toggling airplane mode from the UI, and is much tighter than the individual `svc` subsystem toggles.

## Verification

After the change lands we can observe the next N pull_request-triggered + merge-queue + cron runs to confirm the release job goes from ~flaky-1-in-N to consistently green.

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/EXP-6819)

--- a/experimenter/tests/integration/nimbus/android/test_fenix_enrollment.py
+++ b/experimenter/tests/integration/nimbus/android/test_fenix_enrollment.py
@@ -31,8 +31,9 @@ def test_fenix_enrollment(
     # Prevent the real Nimbus RS fetch from overwriting our local enrollment
     # (without this, Nimbus evolves against the fresh fetch — which does not
     # contain our test experiment — and unenrolls us).
-    subprocess.check_call(["adb", "shell", "svc", "wifi", "disable"])
-    subprocess.check_call(["adb", "shell", "svc", "data", "disable"])
+    subprocess.check_call(
+        ["adb", "shell", "cmd", "connectivity", "airplane-mode", "enable"]
+    )
 
     subprocess.check_call(
         [

